lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -4412,6 +4412,30 @@ would appear as ``$options.vcf_output`` having the values ``'--vcf'`` when true
       </data>
     </outputs>
 ```
+
+Note that variables that correspond to optional select parameters are `None` if nothing is selected. 
+Therefore a filter for such a variable looks like the following example.
+
+### Example
+
+```xml
+    <inputs>
+       <param name="output_type" type="select" optional="true">
+            <option value="save_phase">Phase Movie</option>
+            <option value="save_period">Period Movie</option>
+        </param>
+    </inputs>
+
+    <outputs>
+        <data name="phase_out" format="tiff">
+            <filter>output_type and "save_phase" in output_type</filter>
+        </data>
+        <data name="period_out" format="tiff" label="${movie.name[:-4]}_period">
+            <filter>outputs and "save_period" in outputs</filter>
```

Comment:
```suggestion
            <filter>output_type and "save_period" in output_type</filter>
```

Reply:
This might also be an opportunity to show/explain that filters are applied successively: 

```
<filter>output_type</filter>
<filter>"save_period" in output_type</filter>
```

Reply:
You mean the suggestion of @nsoranzo ? The parameter is called `output_type` (not `output`).

If you mean my comment then you might go: 

```
<filter>output_type is not None</filter>
<filter>"save_period" in output_type</filter>
```

mentioning that filters are called successively and then tell the reader that an equivalent short form is: 

```
<filter>output_type and "save_period" in output_type</filter>
```

Reply:
The name if this parameter is totally free from my understanding